packages/mnemonic-seed-manager/__tests__/localAgent.test.ts

```diff
@@ -15,14 +13,13 @@ import { OrPromise } from '@veramo/utils'
 jest.setTimeout(30000)
 
 const KMS_SECRET_KEY = 'd17c8674f5db9396f8eecccde25e882bb0336316bc411ae38dc1f3dcd7ed100f'
-let databaseFile = 'database.sqlite'
 let dbConnection: OrPromise<DataSource>
 let agent: any
 
 const setup = async (): Promise<boolean> => {
   const db: OrPromise<DataSource> = new DataSource({
     type: 'sqlite',
-    database: databaseFile,
+    database: ':memory:',
```

Comment:
Not sure whether I like this change, as it could just be masquerading stuff. Why is this part of the fix?

Reply:
We have packages using an in-memory database and other ones using a file. I've changed it for the sake of consistency, to avoid to have a database file in the repository and to spare some milliseconds running the tests.

Reply:
DB file should just be in .gitignore if you worry about that.